airflow/utils/cli.py

```diff
@@ -140,10 +141,17 @@ def _build_metrics(func_name, namespace):
     :return: dict with metrics
     """
     sub_commands_to_check = {"users", "connections"}
```

Comment:
Let us add "variables" here

Reply:
Ideally, I have bifurcated both, since these check based upon the flags, and the variables need based upon the keys which is positional args. Adding both at the same place would be wrong, since both need different operations.

Reply:
@amoghrajesh this makes sense.
`sub_commands_to_check` as `sub_commands_to_check_for_sensitive_fields`
Also,
Should we better do `sub_commands_to_check_based_upon_key` to `sub_commands_to_check_for_sensitive_key` since index would be bit in different context?

Reply:
I was thinking index as it will be referred using that. Open to `sub_commands_to_check_for_sensitive_key` to.

Reply:
Yep. Good choices.